PyInstaller/hooks/hook-sqlalchemy.py

```diff
@@ -24,6 +24,10 @@
 # are not. We should explicitly include database backends.
 hiddenimports = ['pysqlite2', 'MySQLdb', 'psycopg2']
 
+# add sqlalchemy.ext.baked to hidden imports
+baked = exec_statement('from sqlalchemy.ext import baked')
+hiddenimports.append(baked)
```

Comment:
Ahh no. Perhaps I shouldn't have mentioned `exec_statement()` to you. This is definitely not what it's for and you don't need it for this hook. `hiddenimports` should just contain the names of modules as `str`s. In this case all you need is:

```suggestion
hiddenimports.append("sqlalchemy.ext.baked")
```